### PR TITLE
Option to expand all collapsed headers #44

### LINKED ISSUE #18: Add an 'expand all weeks' button to the schedule page
We can have `+` and `++` buttons here for expanding/collapsing all weeks in one shot.
image

### LINKED ISSUE #44: Option to expand all collapsed headers
Option to expand all collapsed headers in the schedule. This helps when we need to search the page for a keyword using the Ctrl-F function :)

### PR DESCRIPTION
**Pull request by zeulb**
Fixes #44
Fixed #18 

Added on preference bar

### CODE REVIEW

File: scripts/common.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ function makeAccordion(elementSelector) {
 }
 
 function getContentUsingAjax(section, elementSelector) {
-    pullContent(section, elementSelector, 'Exract from handbook');
+    pullContent(section, elementSelector, 'Extract from handbook');
```
Comment (damithc) [inline scripts/common.js]: Submit this as a separate PR as it is not related to this issue.

**Comment by damithc** [comment]
I would like to see how it looks. Can you push it to `gh-pages` branch and access it from `github.io`. Something like [this](http://wkurniawan07.github.io/website/).

**Comment by zeulb** [comment]
Ok, link to preview is available [here](http://zeulb.github.io/website/)

**Comment by damithc** [comment]
Nice. @wkurniawan07 for your review please

**Comment by wkurniawan07** [comment]
Good to merge :+1:

**Comment by damithc** [comment]
I think something is missing here. Shouldn't there be a change in an html file, to add the buttons and the check boxes?

**Comment by zeulb** [comment]
The buttons are added dynamically by the javascript. But now that I think about it, the code might look neater if the buttons are added statically on the html file. Should I submit another pull request ?

**Comment by damithc** [comment]
The code you have in the preview is different from the one in the PR. One change (the change that uncomment the checkboxes) has been left out of the PR. So the collapse/expand all buttons don't appear at all.

**Comment by zeulb** [comment]
Yes, I didn't uncomment the preference bar on my pull request, I thought it was purposely hidden.

**Comment by damithc** [comment]
It was. But the new buttons appear in the same area as the preference bar right? So without the preference bar, buttons don't appear? If that is the case, we need to enable the preference bar as well.

**Comment by zeulb** [comment]
Yes, it only appear in the preference bar.

**Comment by damithc** [comment]
Can create another PR to enable the preference bar?

**Comment by zeulb** [comment]
Submitted
